### PR TITLE
Fix sample_network_auditing.py so it works with Python3

### PR DESCRIPTION
**Pull request by cforger**
- Added detect for Python 2/3 so the correct iteritems()/iter() is called, but without needing to install six
- Switch to 'in' which works in 2/3 instead of legacy '.has_key()'
- Renamed 'id' variable use (which is a built-in) to my_id.
- Small formatting changes to make it more PEP compliant.

### CODE REVIEW

File: bindings/python/sample_network_auditing.py [hunk 1]
```diff
@@ -7,7 +7,14 @@
...
 import uuid
-import random
+
+# Python 2/3 Compat without installing six
+DEAD_PYTHON = False
+import sys
+
+if sys.version_info[0] < 3:
+    DEAD_PYTHON = True
+    print("Python 2.x is dead. Start thinking about migrating to 3.x. https://wiki.python.org/moin/Python2orPython3")
```
Comment (aquynh) [inline bindings/python/sample_network_auditing.py]: would you mind removing the above line, or commenting it out?

Comment (lunixbochs) [inline bindings/python/sample_network_auditing.py]: I've spent more time reading this PR than any computer will ever waste in `iteritems()` when running this sample. Just switch to `items()` for Python 2.x and drop the DEAD_PYTHON checks.

**Comment by cforger** [comment]
Yes, I can take out the comment. :)

I can also kill the check entirely, and just use items() but I wasn't sure how far back into Python history you wanted to go.

**Comment by lunixbochs** [comment]
I'm pretty sure items long predates iteritems.

**Comment by aquynh** [comment]
merged, thanks!

**Pull request merged**